src/wallet/wallet.cpp

```diff
@@ -3956,7 +3958,10 @@ ScriptPubKeyMan* CWallet::AddWalletDescriptor(WalletDescriptor& desc, const Flat
     auto spk_man = GetDescriptorScriptPubKeyMan(desc);
     if (spk_man) {
         WalletLogPrintf("Update existing descriptor: %s\n", desc.descriptor->ToString());
-        spk_man->UpdateWalletDescriptor(desc);
+        if (auto res = spk_man->UpdateWalletDescriptor(desc); !res)
+        {
```

Comment:
Same as above; would be good to not mix coding styles. No jump line after the condition statement.

And the `res` variable is shadowing another variable in this function. Would be better to rename it to `res_spkm` or `spkm_res`.